go/libkb/api.go

```diff
@@ -620,12 +620,12 @@ func (a *InternalAPIEngine) PostJSON(arg APIArg) (*APIRes, error) {
 	return a.DoRequest(arg, req)
 }
 
-// PostResp performs a POST request and returns the http response.
+// postResp performs a POST request and returns the http response.
 // The returned response, if non-nil, should have
 // DiscardAndCloseBody() called on it.
-func (a *InternalAPIEngine) PostResp(arg APIArg) (*http.Response, error) {
+func (a *InternalAPIEngine) postResp(arg APIArg, sendJSON bool) (*http.Response, error) {
```

Comment:
I see that you're just plumbing this boolean that already exists, so maybe not worth changing in this pr, but: Would it be cleaner if we just looked at `arg.JSONPayload != nil` instead of passing around this `sendJSON` bool?